src/main/java/gololang/Predefined.java

```diff
@@ -534,15 +534,18 @@ public static void textToFile(Object text, Object file, Object charset) throws T
       System.out.write(str.getBytes(encoding));
     } else {
       Path path = pathFrom(file);
-      if (path.getParent() != null) {
-        Files.createDirectories(path.getParent());
-      }
-      Files.write(
+      if (path != null) {
+        Path parent = path.getParent();
+        if (parent != null) {
+          Files.createDirectories(parent);
+        }
+        Files.write(
           path,
           str.getBytes(encoding),
           StandardOpenOption.WRITE,
           StandardOpenOption.CREATE,
           StandardOpenOption.TRUNCATE_EXISTING);
+      }
```

Comment:
no `else`? What if `path` is `null`? (event if I don't know in which case :smile:)